package/upgrade/upgrade_manifests.sh

```diff
@@ -525,6 +557,13 @@ upgrade_addons()
   fi
 }
 
+reuse_vlan_cn() {
+  [[ $UPGRADE_PREVIOUS_VERSION != "v1.0.3" ]] && return
+
+  # delete finalizer
+  kubectl get clusternetwork vlan -o yaml | yq '.metadata.finalizers = []' | kubectl apply -f -
```

Comment:
suggest to:
check `kubectl get clusternetwork vlan -o yaml | yq '.metadata.finalizers = []'`  if `finalizers` is there,
and echo `will delete finalizer`, 
finally remove it
`kubectl get clusternetwork vlan -o yaml | yq '.metadata.finalizers = []' | kubectl apply -f -`

Reply:
The finalizer is always existing in v1.0.3.